tool/tbot/main.go

```diff
@@ -377,9 +378,9 @@ func handleSignals(log logrus.FieldLogger, cancel context.CancelFunc, reloadCh c
 }
 
 func setupLogger(debug bool, format string) error {
-	level := logrus.InfoLevel
+	level := slog.LevelInfo
 	if debug {
-		level = logrus.DebugLevel
+		level = slog.LevelDebug
```

Comment:
Slightly below this, `tbot` handles switching between JSON and Text formatting - does this need adjusting for slog ?

Reply:
Looks good to me 🖖